android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/session/SignInSessionManager.kt

```diff
@@ -0,0 +1,28 @@
+package io.logto.sdk.android.auth.session
+
+import android.annotation.SuppressLint
+import android.net.Uri
+
+object SignInSessionManager {
```

Comment:
Actually do we need this extra SignInSessionManager layer? Is it possible to create a private `SignInSession` instance within the `LogtoClient` itself?

Reply:
SignInSessionManager is a global singleton. `LogtoClient` is not global.
We need to use SignInSessionManager to hold the auth context.
It's equivalent to the `AuthManager` in the old code.